mmv1/products/gkehub2/Feature.yaml

```diff
@@ -274,6 +277,7 @@ properties:
                   - !ruby/object:Api::Type::String
                     name: version
                     description: 'Version of ACM installed'
+                    deprecation_message: The configmanagement.config_sync.oci.version field is deprecated and will be removed in a future major release. Please use configmanagement.version field to specify the version of ACM installed instead.
```

Comment:
Can you add `` around fields configmanagement.config_sync.oci.version and configmanagement.version?